### ISSUE TITLE
AutoInit flag

### ISSUE THREAD

**Issue by weisborg** [dangrossman/daterangepicker#1772]
There needs to be some type of flag to determine if the library initializes with a value or not.  I need to monitor for changes to prompt the user that there are unsaved changes, but since updated by JavaScript it doesn't trigger change unless autoUpdateInput is set to true.  When it is set to true it also sets the dates to current date along with a premature change trigger so my form thinks data changed.  I'd like to leave them blank and not trigger it on init.  Other libraries either have a flag or a date you pass to init with.

Currently I'm just clearing the input values after calling daterangepicker() and clearing my dirtyForm flag, but I'm worried there could be timing issues with the event handlers possibly firing after and setting the dirtyFlag back.  Whenever I do bandaids like this they always seem to bite me later when more code is added and suddenly the timing is off.

**Comment by dangrossman** [comment]
This is a 6 year old library and I've never heard a request quite like this. If you don't want it to populate your form input, then set `autoUpdateInput` to false and populate it yourself when a new selection is made. Or, edit the code. It's free open source. I don't plan on maintaining a new flag for one person, sorry.

**Comment by weisborg** [comment]
It is not the populating I need, it is the populating that is the bug! Having AutoUpdateInput to true is the only way to make it trigger a change event when a date is selected so other events can monitor for changes. YOUR bug is that you made one option do two things. When binding to an empty input you should not be setting it to current date and triggering a change event. The user never changed it!  It should be two options. One for if to init to a current date or some other date. And another to trigger change events...which really shouldn't even be tied to an option and just built in behavior so things get notified of changes since picking a date otherwise changes the value for the input by JavaScript which triggers no event.

**Comment by Biscuri** [comment]
Sorry for digging this, I think it's better than opening a repeated issue. I just updated from an old version, where it worked exactly like @weisborg described, without any flags, the date wasn't init, I set a "value" attribute on input if I needed to, and the update occurred on change event. Making the ```autoUpdateInput``` flag to populate both on init and update makes no sense at all.

I'm using the library on the most basic ways: a field on a form for inserting/updating a row, and another field  on a list to filter the rows by date. The filter and the insert form should not be init with a date, and the updating form should be init with the saved date. I'm sure we are not the only ones using the library that way, and surprised to see that this issue closed without solution and no one made a pull request also.
